tests/roots/test-base/target/edgecase_model_as_attr.py

```diff
@@ -14,7 +14,7 @@ class Model(BaseModel):
     field2: str
     """Field2 Doc String"""
 
-    @validator("field")
+    @field_validator("field")
```

Comment:
I was taking a quick look and noticed that the import is missing being updated as well, same in some other files in the tests (and some docs)